pkg/controllers/interruption/suite_test.go

```diff
@@ -113,13 +115,18 @@ var _ = Describe("InterruptionHandling", func() {
 				ProviderID: fake.RandomProviderID(),
 			},
 		})
+		metrics.NodeClaimsDisruptedCounter.Reset()
 	})
 	Context("Processing Messages", func() {
 		It("should delete the NodeClaim when receiving a spot interruption warning", func() {
 			ExpectMessagesCreated(spotInterruptionMessage(lo.Must(utils.ParseInstanceID(nodeClaim.Status.ProviderID))))
 			ExpectApplied(ctx, env.Client, nodeClaim, node)
 
 			ExpectSingletonReconciled(ctx, controller)
+			ExpectMetricCounterValue(metrics.NodeClaimsDisruptedCounter, 1, map[string]string{
+				"type":     "spot_interruption_kind",
```

Comment:
Can we look at calling this `reason`. I'd like to have us mark the reasons behind why we are disrupting things and then just align the lingo across the codebase.